aiida/common/pluginloader.py

```diff
@@ -125,15 +125,19 @@ def get_plugin(category, name):
 
     return plugin
 
-def load_plugin_safe(base_class, plugins_module, plugin_type):
+def load_plugin_safe(base_class, plugins_module, plugin_type, nodeType, nodePk):
```

Comment:
all good, but I just realised that you should use names_with_underscores for variables instead of camelCase, or we risk that people get confused when specifying the variable names of a function (the same inside the functions).